internal/commands/push.go

```diff
@@ -2,26 +2,34 @@ package commands
 
 import (
 	"bytes"
+	"compress/gzip"
 	"context"
+	"encoding/json"
 	"fmt"
 	"io"
 	"io/ioutil"
 	"os"
 	"strings"
 
+	"github.com/containerd/containerd/content"
```

Comment:
Sounds like a red flag to me that we rely on a containerd package within command line stuff. I hardly can consider containerd as a "utility library" we can cherry-pick commodity functions from.